test/common/router/header_formatter_test.cc

```diff
@@ -1252,14 +1248,14 @@ match: { prefix: "/new_endpoint" }
   - header:
       key: "x-client-ip"
       value: "%DOWNSTREAM_REMOTE_ADDRESS_WITHOUT_PORT%"
-    append: true
+    append_action: APPEND_IF_EXISTS_OR_ADD
   - header:
       key: "x-client-ip-port"
       value: "%DOWNSTREAM_REMOTE_ADDRESS%"
   - header:
       key: "x-client-port"
       value: "%DOWNSTREAM_REMOTE_PORT%"
-    append: true
+    append_action: APPEND_IF_EXISTS_OR_ADD
```

Comment:
There is a lot of test churn. Can you point to the tests that still provide coverage of the deprecated functionality? We should have some until the old field is retired.

Reply:
Yeah, some more test for deprecated `append` is on the way.